README.md

```diff
@@ -46,13 +46,13 @@ Add OnchainKit to your project, install the required packages.
 
 ```bash
 # Use Yarn
-yarn add @coinbase/onchainkit viem@2.x react@18 react-dom@18
+yarn add @coinbase/onchainkit viem@2.x react@18 react-dom@18 graphql@14 graphql-request@6
```

Comment:
So I am been thinking on this.

And as `graphql@14 graphql-request@6` is required only for one utility, I think is best we mention to install that only in the page related to EAS.

A bit like some dependencies are common for the majority of the library, others are just for a few.

Than one that are for a few we explain to add them in their section.

Reply:
Actually let me see if I can make this a bit cuter